File: composeApp/src/commonMain/kotlin/com/hhp227/concafe/presentation/main/cafemanagement/banner/BannerEvent.kt
package com.hhp227.concafe.presentation.main.cafemanagement.banner

sealed interface BannerEvent {
    data object NavigateBack : BannerEvent
    data class NavigateToBannerEdit(
        val cafeId: String?,
        val bannerId: String? = null
    ) : BannerEvent
    data class ShowMessage(val message: String) : BannerEvent
}

File: iosApp/iosApp/Presentation/Main/CafeManagement/Banner/BannerEvent.swift
//
//  BannerEvent.swift
//  ConCafe
//
//  Created by 홍희표 on 2026/03/15.
//

import Foundation

enum BannerEvent {
    case navigateBack
    case navigateToBannerEdit(cafeId: String?)
    case showMessage(String)
}
